### ISSUE TITLE
issue for automatic audio captioning

### ISSUE THREAD

**Issue by Liwen4567** [XinhaoMei/WavCaps#20]
Dear author, hello！
I am trying to use your model for automatic audio captioning, but I got some questions I was used train.py to train 10 epochs from scratch, but I found that CNN14-BART could generate captioning normally, while HTSAT-BART could only generate repeated captioning that had nothing to do with real captioning. May I ask why?

**Comment by Liwen4567** [comment]
The captions generated by HTAST-BART are basically repetitive and have nothing to do with the actual captions。I didn't change the schema or hyperparameters of any of the models.
![image](https://github.com/XinhaoMei/WavCaps/assets/51770083/2563879f-4c4a-407f-a7d1-3c41359c5e2c)

**Comment by XinhaoMei** [comment]
Hello,

If you train the model from scratch, I guess you might need to tune the hyperparameters (i.e., learning rate) carefully.

**Comment by Liwen4567** [comment]
您好！邮件已收到，我将尽快处理。此致      祝好

**Comment by Liwen4567** [comment]
Thank you for your reply！
I have been able to generate normal captions by importing your pre-trained weights, but if I try to start over a lot of times, 
each time, there was no way to generate normal captions. I only used the Clotho.
As you said, the hyperparameters need to be carefully adjusted. Does this make a difference？

**Comment by XinhaoMei** [comment]
Of course, training from scratch and fine-tuning usually requires different learning rates.

**Comment by Liwen4567** [comment]
Thanks! I will try it by different learning rates.

**Comment by zhangron013** [comment]
> Thanks! I will try it by different learning rates.

hi, have you tried other learning rates? What learning rate is required for HTSAT to generate captions normally?
